quickfixj-core/src/main/java/quickfix/mina/ssl/SSLFilter.java

```diff
@@ -19,70 +19,17 @@
 
 package quickfix.mina.ssl;
 
-import java.net.InetSocketAddress;
-import java.net.SocketAddress;
-import javax.net.ssl.SSLContext;
-
-import javax.net.ssl.SSLException;
-import org.apache.mina.core.filterchain.IoFilterChain;
-import org.apache.mina.core.session.IoSession;
 import org.apache.mina.filter.ssl.SslFilter;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.SSLContext;
 
 /**
  * An extended SSL filter based on MINA {@link SslFilter} that applies
  * some adaptations.
  */
 public class SSLFilter extends SslFilter {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());

```

Comment:
In theory this class is no longer required as there is no custom functionality here.

Reply:
Feel free to remove it.

Reply:
I just did.